.yarnrc.yml

```diff
@@ -0,0 +1,5 @@
+nodeLinker: node-modules
```

Comment:
You should copy the plugin for disabling PnP patching of TypeScript and others: https://github.com/microsoft/rnx-kit/pull/2315

Reply:
Done